### ISSUE TITLE
Wrong size of the onscreen keyboard after rotation under PIXEL desktop (image - 2022-04-01)

### ISSUE THREAD

**Issue by karlitos** [cutiepi-io/cutiepi-shell#29]
When I go to the PIXEL desktop, display the virtual keyboard (e.g. opening the terminal) and rotate the screen, the keyboard has wrong size - the keys are huge and and only a part of the keyboard is visible. Can reproduce when rotating form portrait do landscape and also vice versa.

**Comment by penk** [comment]
@karlitos thanks for the report, just quick check for one thing: if you hide the keyboard then click on the input area (e.g. terminal) does it resize correctly? if so it's a window resizing issue.

**Comment by karlitos** [comment]
Yes, rotating the screen, taping to the desktop to *hide* the keyboard and taping to the terminal window again displays the virtual keyboard with the right proportions. I basically just flashed the 2022-04-01 image and encountered this right away.

**Comment by penk** [comment]
Ok noted, I know where to fix it now. Thanks again. :-)
